### PR TITLE
Fix Table to Table integration tests

### PR DESCRIPTION
**Pull request by ryantimjohn**

### CODE REVIEW

**Comment by Brunope** [comment]
Some extra context regarding timestamps:

Both redshift and postgres use a TIMESTAMP WITH TIMEZONE data type, which
can be referred to by an alias TIMESTAMPTZ. It appears if you ask redshift
for its column type, you get TIMESTAMPTZ, but if you ask postgres, you get
TIMESTAMP WITH TIME ZONE.

File: .circleci/config.yml
Comment (Brunope) [reply .circleci/config.yml:617]: Good idea thanks for pointing it out

**Comment by Brunope** [comment]
`AssertionError: Could not find column types filed under ('vertica', 'postgresql') or either individually: ['BIGINT', 'VARCHAR(3)', 'VARCHAR(3)', 'VARCHAR(1)', 'VARCHAR(1)', 'VARCHAR(3)', 'VARCHAR(111)', 'DATE', 'TIME', 'TIMESTAMP', 'TIMESTAMP']
`
Test expects:

```
('vertica', 'postgresql'): [
        'BIGINT', 'VARCHAR(3)', 'VARCHAR(3)', 'VARCHAR(1)', 'VARCHAR(1)',
        'VARCHAR(3)', 'VARCHAR(111)', 'DATE', 'TIME WITHOUT TIME ZONE',
        'TIMESTAMP WITHOUT TIME ZONE', 'TIMESTAMP WITH TIME ZONE'
    ],
```

records-mover is converting time formats that include a timezone and formats that don't include a time zone into one TIMESTAMP. I don't know enough about databases and time formats to know if this is a real problem or not but it seems odd and I'm worried that this conversion might lose some information. @ryantimjohn any input?

**Comment by Brunope** [comment]
After looking more at the tests, I think I'm more confident now that the timestamp conversion is ok. The tests grab the resulting time from the target database and assert that it does match the original expected time, so it must be getting converted properly when going from a timestamp with a time zone to one without.

**Comment by Brunope** [comment]
Gonna squash all these changes down into 1 commit probably before merging

**Comment by ryantimjohn** [comment]
> Gonna squash all these changes down into 1 commit probably before merging

@Brunope sounds good! You have my approval; just because I started this PR before you took it over, you'll have to be the one to approve before merging. Sorry about that.

**Review by Brunope — APPROVED** [review]

**Pull request merged**
